rir/src/compiler/test/PirTests.cpp

```diff
@@ -724,11 +724,16 @@ static Test tests[] = {
                  "theFun <- function() {a<-FALSE; if (a) {q <-1} else "
                  "{if (a) q <- 3 else q <- 2}; q}");
          }),
-    Test("merge_missing",
-         []() {
-             return hasLoadVar("theFun <- function(a) {if (a) {q <-1} else {if "
-                               "(a) 3 else q <- 2}; q}");
-         }),
+    /*
+          // it was removed because with the 'branch elim' optimization
+          // the LoadVar is not present anymore
+          Test("merge_missing",
+           []() {
+               return hasLoadVar("theFun <- function(a) {if (a) {q <-1} else {if
```

Comment:
try changing the test to `function (a,b) if (a) ... else if (b) ...`

Reply:
Yes, it's working this way. I will commit the changes